participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/SiteResponse.java

```diff
@@ -20,8 +20,11 @@
 
   private String siteId;
 
-  public SiteResponse(String siteId, MessageCode messageCode) {
+  private String siteName;
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Updated file to `2020-2021`.